src/platform/iotjs_module_uart-posix-general.inl.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#ifndef IOTJS_MODULE_UART_POSIX_GENERAL_INL_H
+#define IOTJS_MODULE_UART_POSIX_GENERAL_INL_H
+
+#include <errno.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "module/iotjs_module_uart.h"
+
+static iotjs_jval_t jthis;
+static int fd;
+static uv_poll_t poll_handle;
```

Comment:
On nuttx, there are many UART. If user creates more than one UART, this global variables make a risk of collision.